src/components/layout/dashboard/sideNav/ItemBase.tsx

```diff
@@ -89,7 +89,12 @@ const ItemBase = <T extends object>({
                 }
               : undefined,
           backgroundColor: (theme) =>
-            isSelected ? lighten(theme.palette.primary.light, 0.9) : undefined,
+            isSelected
```

Comment:
let's discuss where is a good common place for this logic to live, maybe in the theme?